src/main/java/io/snyk/jenkins/steps/SnykBuildStep.java

```diff
@@ -34,6 +34,7 @@
 import io.snyk.jenkins.SnykReportBuildAction;
 import io.snyk.jenkins.credentials.SnykApiToken;
 import io.snyk.jenkins.tools.SnykInstallation;
+import io.snyk.jenkins.trasnform.ReportConverter;
```

Comment:
Small typo in package name

Reply:
shame on me, fixed with 65db9ca3da8e45cf512ff022c6bf5ba2d27a914e